src/Reader/Feed/AbstractFeed.php

```diff
@@ -236,7 +238,7 @@ public function valid()
         return 0 <= $this->entriesKey && $this->entriesKey < $this->count();
     }
 
-    public function getExtensions()
+    public function getExtensions(): array
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.

Reply:
probably you missed this one 🙈